Documentation/tutorial/intro_overwrite_files.md

```diff
@@ -0,0 +1,110 @@
+Overwrite Files
+===============================
+
+Introduction
+--------------
+DocFX supports processing Markdown files, as well as structured data model in YAML or JSON format.
+
+We call Markdown files *Conceptual File*s, and the structured data model files *Metadata File*s.
+
+Current supported *Metadata File*s include:
+1. YAML files presenting managed reference model following [Metadata Format for .NET Languages](../spec/metadata_dotnet_spec.md).
+2. Swagger JSON files presenting Swagger REST API model following [Swagger Specification Version 2.0](http://swagger.io/specification).
+
+Inside DocFX, both *Conceptual File*s and *Metadata File*s are represented as *Model*s with different properties. Details on *Model* structure for these files are described in [Data model inside DocFX](#data-model-inside-docfx) section.
+
+DocFX introduces the concept of *Overwrite File* to modify or add properties to *Model*s without changing the input *Conceptual File*s and *Metadata File*s.
+
+The format of *Overwrite File*s
+-----------------
+*Overwrite File*s are Markdown files with multiple *Overwrite Section*s starting with YAML header block. A valid YAML header for an *Overwrite Section* *MUST* take the form of valid [YAML](http://www.yaml.org/spec/1.2/spec.html) set between triple-dashed lines and start with property `uid`. Here is a basic example of an *Overwrite Section*:
+
+```md
+---
+uid: microsoft.com/docfx/Contacts
+---
+Further description for `microsoft.com/docfx/Contacts`
+```
+
+Each *Overwrite Section* is transformed to *Overwrite Model* inside DocFX. For the above example, the *Overwrite Model* represented in YAML format is:
+
+```yaml
+uid: microsoft.com/docfx/Contacts
+some_property: value
+conceptual: <p><b>Content</b> in Markdown</p>
+```
```

Comment:
exactly

---
In reply to: [57170647](https://github.com/dotnet/docfx/pull/233#discussion_r57170647) [](ancestors = 57170647)